src/lib/dnssd/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -319,10 +322,16 @@ void AdvertiserMinMdns::OnQuery(const QueryData & data)
 
 CHIP_ERROR AdvertiserMinMdns::Init(chip::Inet::EndPointManager<chip::Inet::UDPEndPoint> * udpEndPointManager)
 {
+    // TODO: Per API documentation, Init() should be a no-op if mIsInitialized
```

Comment:
its not a no-op with this change because the instance name is going to be changed, right?